tests/exporters/test_onnx.py

```diff
@@ -11,9 +11,24 @@
 from setfit.trainer import SetFitTrainer
 
 
-def test_export_onnx_sklearn_head():
+@pytest.mark.parametrize(
+    "model_path, input_text",
+    [
+        ("lewtun/my-awesome-setfit-model", ["i loved the spiderman movie!", "pineapple on pizza is the worst 🤮"]),
+        (
+            "lewtun/setfit-ethos-multilabel-example",
+            [
+                "Women are made for making babies and cooking dinner and nothing else!!!",
+                "Lmfao that fat slob in the wheelchair",
+                "Indian Ned Flanders. You look like you professionally harrass subpar women on Facebook",
+                "Aren’t you ashamed about your body colour?",
+                "Islam is  the home of terriorism .",
```

Comment:
I understand that the ethos model is a hate-speech detection model, but because we're not actually checking the correctness of the model outputs, I think I would prefer more "normal" sentences here, if that's okay with you :)

Reply:
I'm fairly certain that difference is considerably larger than it should be, indeed. I'd love for you to try and help debug it!

Reply:
It might just be this old, unsolved problem https://github.com/huggingface/setfit/issues/291#issue-1562241805. It's probably because `lewtun/my-awesome-setfit-model` uses mpnet.